environment.yml

```diff
@@ -4,28 +4,30 @@ channels:
  - anaconda
  - defaults
 dependencies:
- - python
+ - python<3.10
  - pip
- - affine
- - boto3
- - fiona
- - gdal
- - geopandas
- - importlib_metadata  # drop when py>=3.8
- - jupyter
- - matplotlib
- - numpy
- - pandas
- - pyproj
- - rasterio
- - requests
- - shapely
- - tqdm
- # for testing and distribution
- - flake8-blind-except
- - flake8-builtins
- - flake8-import-order
- - pytest
- - setuptools
- - setuptools_scm
- - wheel
+ - pip:
+    - affine>=2.3.0
+    - Fiona>=1.8.6
+    - geopandas>=0.5.1
+    - numpy>=1.17.2
+    - pandas>=0.25.1
+    - rasterio>=1.0.28
```

Comment:
Any reason this isn't just `rasterio>=1.2` like in `environments/environment-isce.yml` (and also the inverse; why the constraint there?)

Reply:
Yea - the only problematic version is `gdal` currently. Hopefully that resolves in the not-to-distant future.